### DISCUSSION TITLE
Delete my account

### DISCUSSION THREAD

**Discussion by Shanewade76** [community/community#53276] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I delete my account.

**Comment by radar-blip** [comment, marked as answer]
Go into your `Settings` and in the left column click `Account`
Scroll down and you will see this:

![del](https://user-images.githubusercontent.com/39196767/233268878-2c1cb74b-d62f-402f-86b3-4acf6027351b.png)

**Reply by Shanewade76** [reply]
Thank you.

**Reply by mybossladyla** [reply]
it doesnt let me delete the account

**Reply by michcong** [reply]
where are the settings so I can delete my account

**Reply by radar-blip** [reply]
On the right side of your profile, click your icon. Settings is at the bottom of list.

From there, _Account_ is on the left side, click on that and scroll down.

**Reply by michcong** [reply]
Thanks for your reply :)  I can't find my profile.  Where do I look for it?

**Reply by radar-blip** [reply]
https://github.com/michcong

**Reply by adhsoubhaya-coder** [reply]
Delete my account

**Comment by ghost** [comment]
Thank you

**Comment by ghost** [comment]
It is expensive for me

**Comment by ghost** [comment]
It is expensive for me and I well active this account next time

**Comment by ghost** [comment]
Non riesco a cancellare account

**Comment by Ferrisomogyi** [comment]
Great, deleten my account does not work. How annoying. Maling hard to lease huh

**Comment by Gmangoodvi** [comment]
Delete my account

**Reply by deletedUser78961** [reply]
I think that a deliberate serious violation of the GitHub terms of use will speed up this process. Will the account be deleted after the ban?

**Comment by sitansulinux-code** [comment]
need to delete this account

**Comment by Kurnia-79** [comment]
# Assist to delete my account
